.travis.yml

```diff
@@ -7,7 +7,7 @@ android:
     - platform-tools
 
 jdk:
-  - oraclejdk8
+  - oraclejdk9
```

Comment:
😮

Reply:
you sure you want to change this?

Reply:
Well, that's why I asked. Also it looks like we'd need to go to 10, 9 isn't even downloadable anymore